### DISCUSSION TITLE
Drawing an ROC for a custom NER

### DISCUSSION THREAD

**Discussion by alpha-1-centauri** [explosion/spaCy#11502] [Help: Best practices]
Hi, I have re-trained my own custom NER, and would like to generate some ROC plots to assess my model.
How can this be done with Spacy?

**Comment by polm** [comment]
An ROC curve is typically plotted by tuning the threshold of a binary classifier, but the NER model doesn't have a threshold, so there's no way to do that.

You could use the spancat component instead, which does have a tunable threshold.
